tripal_chado/src/Plugin/Field/FieldType/ChadoSequenceChecksumTypeDefault.php

```diff
@@ -60,6 +66,19 @@ public static function defaultStorageSettings() {
     return $settings;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public static function generateSampleValue(FieldDefinitionInterface $field_definition) {
+    $value = [];
+
+    $value['record_id'] = 0;
+    $value['seqlen'] = 9;
+    $value['md5checksum'] = md5('AATGCGCTT');
```

Comment:
```suggestion
    $value['seqlen'] = 18;
    $value['md5checksum'] = md5('ACCCGCATTCCGGCGCTG');
```
Translated to protein gives 'TRIPAL' 😁